bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpServerHandler.java

```diff
@@ -207,7 +207,13 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                                 // No parent found, so make it the root directory
                                 browseTarget = DIRECTORY_ROOT;
                             }
-                            currentEntry = parentMap.get(browseTarget);
+                            UpnpEntry entry = parentMap.get(browseTarget);
+                            if (entry == null) {
+                                logger.info("Browse target not found. Exiting.")
```

Comment:
It will build better with:

```suggestion
                                logger.info("Browse target not found. Exiting.");
```

Reply:
obviously.